src/auto/accel_group.rs

```diff
@@ -49,11 +53,11 @@ impl Default for AccelGroup {
 pub trait AccelGroupExt {
     //fn activate<P: IsA<glib::Object>>(&self, accel_quark: /*Ignored*/glib::Quark, acceleratable: &P, accel_key: u32, accel_mods: gdk::ModifierType) -> bool;
```

Comment:
We should probably really add `GQuark` bindings at some point. Easy enough and appearing in so many places of APIs.

Reply:
My next glib PR I suppose. ;)